test/boot_requirements_checker_test.rb

```diff
@@ -40,7 +40,7 @@ def find_vol(mount_point, volumes)
     let(:storage_arch) { instance_double("::Storage::Arch") }
 
     before do
-      Yast::Storage::FakeProbing.new
+      Yast::Storage::StorageManager.fake_from_yaml
```

Comment:
> For cases like this, in which we only want to do the first part but not the second one,
> we should have another method

I can't follow you here. That's like arguing against constructors with arguments.

fake() resp. fake_from_yaml() creates a fake instance (and populates it with the arg)